jsk_denso_robot/sample_pack/canon_camera/package.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0"?>
```

Comment:
please cleanup this file like https://github.com/jsk-ros-pkg/jsk_robot/pull/1302/files#diff-ea80a0e01ea72ead21b8a5b499c1b99d8a99be31646a6af0cea39b0967ddbf8d